pkg/server/log.go

```diff
@@ -0,0 +1,29 @@
+package server
+
+import (
+	"io"
+	"log"
+	"strings"
+)
+
+type FilteredWriter struct {
+	io.Writer
+}
+
+var msgsToFilter = []string{
```

Comment:
If the solution for that annoying problem is specific for AWS ELB, I disagree. As I said [here](https://github.com/luizalabs/teresa-api/pull/220#discussion_r128630487), however, I'll search a generic fix.